internal/mode/static/handler.go

```diff
@@ -72,7 +72,9 @@ type eventHandlerConfig struct {
 // (2) Keeping the statuses of the Gateway API resources updated.
 // (3) Updating control plane configuration.
 type eventHandlerImpl struct {
-	cfg eventHandlerConfig
+	// latestConfiguration is the latest Configuration generation.
+	latestConfiguration *dataplane.Configuration
```

Comment:
Was thinking of a way to get the number of Endpoints. @pleshakov suggested getting it from the Upstreams in the generated configuration. Chose to put something like `latestGraph` but for the configuration on the event handler. Could I get some eyes specifically on this work?

Reply:
It would be nice if we had some sort of hook into the `store` itself to get all the resources we have stored. Having this config on the event handler seems like the wrong place.

Reply:
@pleshakov thoughts?